tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
+USERNAME_GRAFANA = os.environ["USERNAME_GRAFANA"]
+PASSWORD_GRAFANA = os.environ["PASSWORD_GRAFANA"]
+SUCCESS_SIGN = "✅"
+ERROR_SIGN = "❌"
+
+def pd_list_users():
```

Comment:
Probably a good idea to indicate that this method creates grafana users (e.g. `create_grafana_users` or `migrate_users`).

Reply:
> Probably a good idea to indicate that this method creates grafana users (e.g. `create_grafana_users` or `migrate_users`).

In this commit, I changed the function name to follow the default.
https://github.com/grafana/oncall/pull/403/commits/a920062c8faea39f95348d131b5b016a8dff62ab